doc/license.md

```diff
@@ -77,8 +77,7 @@ should be copied to `./plugins.cfg` before calling `./configure-plugins` to take
 
 ### Less restrictive than LGPL
 
-* libr/asm/arch/gnu: GPLv2
-* libr/bin/mangling/cxx: GPLv2
+* libr/bin/mangling/cxx: GPLv2 or later, with linking exception
```

Comment:
```suggestion
* libr/asm/arch/*/gnu: GPLv2
* libr/bin/mangling/cxx: GPLv2 or later, with linking exception
```

Reply:
GPLv2 is obviously *not* "Less restrictive than LGPL". The code in */gnu has no linking exception, AFAICS.

Reply:
Correct, no linking exception in binutils